kitsune/sumo/static/sumo/js/device-migration-wizard.js

```diff
@@ -122,21 +118,78 @@ class ConfigureStep extends BaseFormStep {
     return `
       <template>
         <div>
-          <p>This is where the sync step will go.</p>
-          <p id="sync-status"></p>
+          <p id="header">
+            <img class="icon" src="${infoImageURL}" aria-hidden="true"></img>
+            <span>${gettext("You are now logged in to Firefox Accounts")}</span>
+            <a href="#" data-event-category="device-migration-wizard" data-event-action="click" data-event-label="forgot-password">${gettext("Forgot password?")}</a>
+          </p>
+          <p id="sync-status-container">
+            <img class="icon" src="${syncingImageURL}" aria-hidden="true"></img>
+            <span><strong>${gettext("Syncing:")}</strong></span>
+            <span id="sync-status"></span>
+          </p>
+          <ul id="instructions">
+            <li>${gettext("We may not need to say that it’s “off” at the beginning, but rather a “get set up” - type of introduction")}</li>
+            <li>${gettext("Make sure there ‘s context for how it works - ex/ timing, frequency, limitations")}</li>
+            <li>${gettext("set expectations we sync all data across all devices - can’t pick and choose")}</li>
+          </ul>
+          <p id="buttons">
+            <button id="turn-on-sync" data-event-category="device-migration-wizard" data-event-action="click" data-event-label="turn-on-sync">${gettext("Turn on sync")}</button>
+            <button id="change-sync-prefs" data-event-category="device-migration-wizard" data-event-action="click" data-event-label="change-sync-prefs">${gettext("Change what you are syncing")}</button>
+            <button id="next" data-event-category="device-migration-wizard" data-event-action="click" data-event-label="configuration-next">${gettext("Next")}</button>
+          </p>
         </div>
       </template>
     `;
   }
 
+  get styles() {
+    let linkEl = document.createElement("link");
+    linkEl.rel = "stylesheet";
+    linkEl.href = configurationStepStylesURL;
+    return linkEl;
+  }
+
+  constructor() {
+    super();
+    let buttons = this.shadowRoot.querySelector("#buttons");
+    buttons.addEventListener("click", this);
+  }
+
   render(prevState, state) {
     if (this.state.syncEnabled !== prevState.syncEnabled) {
       let statusEl = this.shadowRoot.getElementById("sync-status");
       statusEl.textContent = this.state.syncEnabled
-        ? "Syncing: ON"
-        : "Syncing: OFF";
+        ? gettext("ON")
+        : gettext("OFF");
```

Comment:
Do we want to do this via styles / `text-transform: uppercase` instead? Seems slightly safer than relying on translations being all caps.

Reply:
@hannajones Good catch!

Reply:
Oh, that's a great idea. Thanks, will do before landing!